mindsdb/integrations/handlers/tpot_handler/requirements.txt

```diff
@@ -0,0 +1,2 @@
+tpot
```

Comment:
I recommend bounding the major version, at least. This will ensure the handler doesn't suddenly break if there are breaking new releases.